spark-connector/spark-connector-runtime/build.gradle.kts

```diff
@@ -28,7 +28,7 @@ tasks.withType<ShadowJar>(ShadowJar::class.java) {
   // Relocate dependencies to avoid conflicts
   relocate("com.google", "com.datastrato.gravitino.shaded.com.google")
   relocate("google", "com.datastrato.gravitino.shaded.google")
-  relocate("org.apache", "com.datastrato.gravitino.shaded.org.apache")
+  relocate("org.apache.hc", "com.datastrato.gravitino.shaded.org.apache.hc")
```

Comment:
Do you have other dependencies need to relocate? If you have more, it would better to list one by one and shade them, not blindly shade all starting from org.apache...

Reply:
no other package to shade for now